src/main/java/bio/terra/cli/apps/utils/DockerClientWrapper.java

```diff
@@ -71,10 +75,9 @@ public boolean checkImageExists(String imageId) {
    * @param workingDir the directory where the commmand will be executed
    * @param envVars a mapping of environment variable names to values
    * @param bindMounts a mapping of container mount point to the local directory being mounted
-   * @return id of the container that started
    * @throws SystemException if the local directory does not exist or is not a directory
    */
-  public String startDockerContainer(
```

Comment:
I moved the container id to a property of this class. Previously the methods of this class returned the container id and then had it passed back in for almost all methods. Since the container id was only being used in this class, I just made it a property and changed the usage here to be instance vs static.